core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/impl/ContentUpgradeImpl.java

```diff
@@ -29,6 +29,8 @@
 import javax.jcr.query.Query;
 import javax.servlet.http.HttpServletResponse;
 
+import de.valtech.aecu.core.groovy.console.bindings.actions.resource.*;
```

Comment:
minor: not everybody likes * imports

Reply:
yes I mentioned this also but it looks like the intellij settings file does not prevent it so ok I guess